cmd/prometheus/main.go

```diff
@@ -265,6 +266,9 @@ func main() {
 	a.Flag("query.max-samples", "Maximum number of samples a single query can load into memory. Note that queries will fail if they try to load more samples than this into memory, so this also limits the number of samples a query can return.").
 		Default("50000000").IntVar(&cfg.queryMaxSamples)
 
+	a.Flag("feature.promql.allow-looking-ahead-of-eval-time", "Allow looking ahead of eval time in PromQL. Enables @ modifier in PromQL.").
```

Comment:
This is not what was agreed. This should be a comma-separated list of feature names, and the feature here is the @ modifier.

Also don't conflate looking ahead with the modifier. You can use the modifier without looking into the future.